openmetadata-ui/src/main/resources/ui/src/components/Glossary/GlossaryTerms/tabs/AssetsTabs.component.tsx

```diff
@@ -124,24 +124,25 @@ const AssetsTabs = forwardRef(
       isSummaryPanelOpen,
       onAddAsset,
       onRemoveAsset,
-      assetCount,
       queryFilter,
       isEntityDeleted = false,
       type = AssetsOfEntity.GLOSSARY,
       noDataPlaceholder,
       entityFqn,
+      assetCount,
     }: AssetsTabsProps,
     ref
   ) => {
-    const popupRef = React.useRef<HTMLElement>(null);
     const [itemCount, setItemCount] = useState<Record<EntityType, number>>(
       {} as Record<EntityType, number>
     );
 
-    const [activeFilter, setActiveFilter] = useState<SearchIndex[]>([]);
+    const [activeFilter, _] = useState<SearchIndex[]>([]);
```

Comment:
if we are not using this, then lets remove

Reply:
for now we have hidden, in 1.3 we will remove it.